server/http_handler.go

```diff
@@ -1696,3 +1697,63 @@ func (h dbTableHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	dbTblInfo.DBInfo = dbInfo
 	writeData(w, dbTblInfo)
 }
+
+// testHandler is the handler for tests. It's convenient to provide some APIs for integration tests.
+type testHandler struct {
+	*tikvHandlerTool
+	gcIsRunning uint32
+}
+
+// ServeHTTP handles test related requests.
+func (h *testHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	params := mux.Vars(req)
+	mod := strings.ToLower(params["mod"])
+	op := strings.ToLower(params["op"])
+
+	switch mod {
+	case "gc":
+		h.handleGC(op, w, req)
+	default:
+		writeError(w, errors.NotSupportedf("module(%s)", mod))
+	}
+}
+
+// Supported op:
+//   * resolvelock?safepoint={uint64}&physical={bool}
```

Comment:
Better to add more explanations on this usage so that QA teammates could quickly learn how to use this api